packages/office-ui-fabric-react/src/components/Fabric/Fabric.base.tsx

```diff
@@ -29,7 +29,26 @@ export class FabricBase extends React.Component<
     });
     const divProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(this.props, divProperties);
 
-    return <div {...divProps} className={classNames.root} ref={this._rootElement} />;
+    let fabricComponent;
+    if (this.props.applyTheme) {
+      // apply theme only to the div
+      fabricComponent = (
+        <div
+          {...divProps}
+          style={{ backgroundColor: this.props.theme!.semanticColors.bodyBackground }}
+          className={classNames.root}
+          ref={this._rootElement}
+        />
+      );
+    } else if (this.props.applyThemeToBody) {
+      // apply theme to the body of the whole page
+      document.body.style.backgroundColor = this.props.theme!.semanticColors.bodyBackground;
```

Comment:
no! Please don't. This is a permanent update. What you could do is create a "bodyTheme" class, and then conditionally add/remove it to body.

Reply:
done - did just the conditional add because I think if I also did 
`    else {
      document.body.classList.remove('ms-Fabric-body-themed');
    } ` it might throw errors at not finding that class right (since there's only one case where it would be added)?